applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -17,6 +18,7 @@
 
 #include "macros_common.h"
 #include "nrf5340_audio_common.h"
+#include "bt_mgmt.h"
```

Comment:
Is this needed? None of the other top level stream files include this.

Reply:
Yes and they all do in my version!